mac/deploy_mac.sh

```diff
@@ -181,6 +172,33 @@ build_installer_image() {
         "${deploy_path}/"
 }
 
+build_storesign_pkg() {
+    # Build installer pkg file - for submission to App Store
+    if [[ -z "$macapp_cert_name" ]]; then
+        echo "No macapp certificate specified. Thus skipping pkg creation for App Store"
+        return
+    else
```

Comment:
This can end the `if` as there's been a return - no need for an intented `else` block.
```suggestion
    fi
```

Reply:
Actually, move the whole `if` block out and wrap the call -- then the comment on the call will be accurate and the proc will just do its thing unconditionally.